packages/openneuro-components/src/dataset/DatasetHeader.tsx

```diff
@@ -18,10 +20,16 @@ export const DatasetHeader: React.FC<DatasetHeaderProps> = ({
               <a href={'/' + modality}>
                 <div className="hexagon-wrapper">
                   <div className="hexagon no-modality"></div>
-                  <div className="label">{modality.substr(0, 4)}</div>
+                  <div className="label">
+                    {modality === null ? (
+                      <i className="fa fa-circle-o-notch fa-spin"></i>
+                    ) : (
+                      modality.substr(0, 4)
+                    )}
+                  </div>
                 </div>
               </a>
-              {pageHeading}
+              {renderEditor ? renderEditor() : pageHeading}
```

Comment:
```suggestion
              {renderEditor?.() : pageHeading}
```
The way you did it is great. Another (newish) way to do it is by using the optional chaining operator.